star-common-lib.pretty/PWR163.kicad_mod

```diff
@@ -1,14 +1,14 @@
-(module PWR163 (layer F.Cu) (tedit 5EE57548)
+(module PWR163 (layer F.Cu) (tedit 5F13977C)
   (fp_text reference REF** (at 0.508 -12.192) (layer F.SilkS)
     (effects (font (size 1 1) (thickness 0.15)))
   )
   (fp_text value PWR163 (at 0.508 -10.668) (layer F.Fab)
     (effects (font (size 1 1) (thickness 0.15)))
   )
-  (pad "" smd rect (at 0 -4.2545) (size 7.874 8.509) (layers F.Cu F.Paste F.Mask))
+  (pad 3 smd rect (at 0 -4.2545) (size 7.874 8.509) (layers F.Cu F.Paste F.Mask))
   (pad 2 smd rect (at -2.54 3.683) (size 1.651 3.81) (layers F.Cu F.Paste F.Mask))
   (pad 1 smd rect (at 2.54 3.683) (size 1.651 3.81) (layers F.Cu F.Paste F.Mask))
-  (model /home/neelay/Downloads/pwr163.stp
+  (model /home/neelay/Documents/Calstar/cas-hardware/hardware-sch-blocks/star-common-3d/pwr163.stp
```

Comment:
Unless intentional, I don't think this model reference should be to a local filepath